moz-webgpu-cts/src/main.rs

```diff
@@ -129,14 +132,12 @@ fn main() -> ExitCode {
 
 fn run(cli: Cli) -> ExitCode {
     let Cli {
-        gecko_checkout,
+        browser,
+        checkout,
         subcommand,
     } = cli;
 
-    let gecko_checkout = match gecko_checkout
-        .map(Ok)
-        .unwrap_or_else(search_for_moz_central_ckt)
-    {
+    let checkout = match checkout.map(Ok).unwrap_or_else(search_for_moz_central_ckt) {
```

Comment:
nitpick: Let's use less Firefox-specific language for this, since this is also getting called for Servo.

Pushed as a separate `fix` commit with e86bfd0.